Spigot-Server-Patches/0530-Expose-Arrow-getItemStack.patch

```diff
@@ -0,0 +1,74 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nesaak <52047222+Nesaak@users.noreply.github.com>
+Date: Fri, 22 May 2020 13:40:50 -0400
+Subject: [PATCH] Expose Arrow getItemStack
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityArrow.java b/src/main/java/net/minecraft/server/EntityArrow.java
+index 9c97edf9c9e9a8cdf029264f6b563090142c686b..aaa55c938095f6848d76216fd74d311a48ea71c0 100644
+--- a/src/main/java/net/minecraft/server/EntityArrow.java
++++ b/src/main/java/net/minecraft/server/EntityArrow.java
+@@ -556,7 +556,7 @@ public abstract class EntityArrow extends Entity implements IProjectile {
+         }
+     }
+ 
+-    protected abstract ItemStack getItemStack();
```

Comment:
this patch could be simplified by adding a public obfhelper, would be more future-proof too